itou/approvals/models.py

```diff
@@ -61,6 +61,12 @@ class CommonApprovalMixin(models.Model):
 
     class Meta:
         abstract = True
+        constraints = [
+            models.CheckConstraint(
+                name="%(app_label)s_%(class)s_start_before_end",
+                check=models.Q(start_at__lte=models.F("end_at")),
```

Comment:
Si j'en crois le `clean()` la fin ne peux pas être non-plus le début :
```suggestion
                check=models.Q(start_at__lt=models.F("end_at")),
```

Reply:
ah ça m'embête ça: on a un l'agrément PE pk=398481 où c'est le cas :sweat_smile: .

Et tant que les champs start/end sont inclusifs je trouve cela étonnant d'interdire les PASS de 1 jour (en autorisant 2 jours ).

Reply:
Peut-être à ajouter dans la carte dédiée au fait de mettre au propre les inclusifs/exclusifs pour tout ce qui touche au PASS.

Pour l'agrément PE je pense qu'on peux donner un jour gratuit, surtout que si c'est le seul c'est très surement une erreur lors de la consolidation des agréments et de leur extension/avenants.

Reply:
Je vais passer en strictement supérieur pour éviter les PASS ou start = end vu que coté Pole Emploi ils sont interprétés comme annulés.
Et vu que les Agrément PE sont en lecture seule (et voués à disparaître dans un avenir proche), je leur épargne la contrainte.